Sources/Sentry/Public/SentryHttpInterceptor.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * A class used to automatically track http requests.
+ * You can used it by passing it in you URL Session configuration manually
+ * or by turning on automatic instrumentation.
```

Comment:
I expect Auto instrumentation will be turned on by default (given a sampling rate is defined for tracing).
In which case we need to update the docs to:

```suggestion
 * A class used to automatically track http requests.
```